app/assets/stylesheets/regions/mastheads/slider-masthead.scss

```diff
@@ -29,44 +51,69 @@ div.slider.version-201611 {
       }
 
       &.header {
-        width: 350px;
+        width: 100%;
+        text-align: left;
         color: $white;
+        
+        @include media($medium-screen) {
+          width: 35%;
+          vertical-align: bottom;
+          text-align: center;
+        }
 
         div.aligned {
-          position: absolute;
-          bottom: 12px;
           padding: 32px;
-          width: 100%;
 
-          div.subheader.absent {
-            display: none;
-          }
+          div.subheader.absent { display: none; }
 
           h2 {
             margin: 0px auto 24px;
             padding: 0px;
-            text-align: center;
-            font-size: 40px;
-            line-height: 52px;
+            font-size: 22px;
+            line-height: 30px;
             font-weight: bold;
+
+            @include media($medium-screen) {
+              font-size: 30px;
+              line-height: 40px;
+            }
+
+            @include media($large-screen) {
+              font-size: 40px;
+              line-height: 52px;
+            }
           }
 
           hr {
+            display: none;
+            
+            @include media($medium-screen) {
             /* See: http://stackoverflow.com/a/4151770/6763239 */
-            height: 2px;
-            width: 35%;
-            margin:0 auto;
-            color: white;
-            background-color: white;
-            border: none;
+              display: block;
+              height: 2px;
+              width: 35%;
+              margin:0 auto;
+              color: white;
+              background-color: white;
+              border: none;
+            }
```

Comment:
This looks to me like `hr` line will only show on medium screens and not larger ones (as I would expect). Am I reading this correctly?

Checking the console in Chrome, I see I'm not. `$medium-screen` means 780px or larger.